### PR TITLE
[Zeebe] Removed resource from Deployment:CREATED event

### LINKED ISSUE #2339: Resource in `Deployment:CREATED` event are no longer available in replicated partition from 8.3
Due to [camunda/zeebe#13233](https://github.com/camunda/zeebe/issues/13233) resource are no more available inside `Deployment:CREATED` event. Resource is available in the `Process:CREATED` and the `DecisionRequirements:CREATED` instead. User can no longer consume `Deployment:CREATED` to get the BPMN model.

### PR DESCRIPTION
**Pull request by nicpuppa**
## Description

Closes #2339 
Related [issue](https://github.com/camunda/zeebe/issues/13233)

Removed resource from `Deployment:CREATED` event. Resource is still available in `Process:CREATED` or `DecisionRequirements:CREATED`

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by remcowesterhoud** [comment]
@nicpuppa Could you sort this PR out. It seems it contains commits that don't belong in it and has a conflict 🙂

**Comment by nicpuppa** [comment]
> @nicpuppa Could you sort this PR out. It seems it contains commits that don't belong in it and has a conflict 🙂

Hi @remcowesterhoud, I create this PR on top of [this](https://github.com/camunda/camunda-platform-docs/pull/2324) cause I saw that they already create the `Update 8.2 to 8.3` guide. How we want to proceed ?

**Comment by remcowesterhoud** [comment]
As the PR you linked is on hold, I'd propose to extract your changes and just add the page in this PR. That PR doesn't have to block this one.

File: docs/guides/update-guide/820-to-830.md
Comment (remcowesterhoud) [inline docs/guides/update-guide/820-to-830.md:13]:
```suggestion
### Resources no longer available in the Deployment:CREATED event
```

File: docs/guides/update-guide/820-to-830.md
Comment (remcowesterhoud) [inline docs/guides/update-guide/820-to-830.md:15]:
```suggestion
Resources in the **Deployment:CREATED** event are no longer available. Custom Exporters using the resources from this event must be modified to get them from the **Process:CREATED** event for BPMN models and the **DecisionRequirements:CREATED** event for DMN models.
```

**Review by remcowesterhoud — CHANGES_REQUESTED** [review]
Thanks @nicpuppa, I have some small spelling/grammar changes and I've added some context on when this is relevant for users. Please have a look at my suggestions 👀

**Comment by nicpuppa** [comment]
Thank you @remcowesterhoud for the suggestions 💪

**Review by remcowesterhoud — APPROVED** [review]
LGTM from engineering perspective, please wait for the technical writer review before merging!

File: docs/guides/update-guide/introduction.md
Comment (christinaausley) [inline docs/guides/update-guide/introduction.md:27]: Should we place a hold label until this is released, or create another issue so we don't forget?

**Review by christinaausley — APPROVED** [review]
One non-blocking comment 👍

File: docs/guides/update-guide/introduction.md
Comment (nicpuppa) [reply docs/guides/update-guide/introduction.md:27]: I create this [issue](https://github.com/camunda/camunda-platform-docs/issues/2377)

**Pull request merged**
